velox/common/memory/ByteStream.cpp

```diff
@@ -97,7 +97,8 @@ void ByteInputStream::seekp(std::streampos position) {
     }
     toSkip -= range.size;
   }
-  VELOX_FAIL("Seeking past end of ByteInputStream: {}", position);
+  VELOX_FAIL(
+      "Seeking past end of ByteInputStream: {}", static_cast<int>(position));
```

Comment:
`position` is of type `std::streampos` which can be 64-bit long.
https://stackoverflow.com/questions/12774038/stdstreampos-stdstreamoff-and-stdstreamsize-to-long-long-int
The above link says it is better to use `long long`.
For safety, we can add `static_assert(sizeof(std::streamsize) <= sizeof(long long))`. Since this is a compile-time check, it won't hurt performance.